unabridged-works-volume-1/original/14-boston.adoc

```diff
@@ -2254,6 +2348,9 @@ to which the spirit of the Lord alone can make them willing,
 nor to fall upon them because they were unwilling.
 This is from the dragon, wherever it is found.
 
+//the objection or questioin of for the next answer seems to start more at the end of this paragraph, and I wasn't sure what to do.
```

Comment:
yeah, it's tough. i'm not sure what to do about it either, but i think your approach seems as good as any I can think of right now, so let's stick with it.